docs/src/tutorials/tutorial_5.md

```diff
@@ -351,7 +351,7 @@ my_scc_calculation (generic function with 1 method)
 
 Now that we have our post-trial function, we can proceed to obtain our two models and run the simulation. Note that we are using a Mimi MarginalModel `mm` from MimiDICE2010, which is a Mimi object that holds both the base model and the model with the additional pulse of emissions.
 
-```jldoctest tutorial4; output = false, filter = r".*"s
+```julia
```

Comment:
removed the dockets for now because MimiDICE2010 uses .marginal instead of .modified, can re-add it once the correct version of that model is used